src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let clone = { ...state };
+  const result = [];
```

Comment:
giving your vars abstract names is awesome, but not this much, it may confuse other reviewers
generally, avoid giving names containing `result`, `res`, `arr`, `st`, instead do something like this – `arrayOfStates`, `stateClone`, etc.

Reply:
renamed  clone to stateClone